pyslackers_website/marketing/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.conf.urls import url
 
-from .views import Index, TermsOfService
+from .views import Index, TermsOfService, info
 
 
 urlpatterns = [
     url(r'^$', Index.as_view(), name='index'),
     url(r'^terms/$', TermsOfService.as_view(), name='tos'),
+    url(r'^info/$', 'info', name="info")
```

Comment:
Please keep consistent use of quotes in the application (note that everywhere is using single quotes: `'`). I think I would prefer this be `/version` as well, since I think `/info` may later be used to provide a lot more.

I am also not confident that the quoting of the view name import works as you want, from the linting perspective it will appear as an unused import.